tools/make_hgrid/make_hgrid.c

```diff
@@ -375,7 +426,7 @@ void fill_cubic_grid_halo(int nx, int ny, int halo, double *data, double *data1_
     }
 
     for(i=1; i<=nxp; i++) {
-      data[i] = data2_all[ls*nxp*nyp+(nxp-i)*nyp+(nx-1)]; /*south */
+      data[i] = data2_all[ls*nxp*nyp+(nxp-i)*nxp+(nx-1)]; /*south */
```

Comment:
Was this, and the other two on line 440 and 445 errors in the original version?

Reply:
Thanks for pointing this out. After investigating commit histories here and in the HAFS-community git repo (which is where I originally obtained the code for make_hgrid), my guess is that the changes I have are probably outdated.

**Explanation:** 
It looks like the version I have is similar to what was in the NOAA-GFDL/FRE-NCtools master branch _prior_ to edits in September 2015 by Zhi Liang, which _appear_ to be bug fixes (see lines 363, 374, 379 at https://github.com/NOAA-GFDL/FRE-NCtools/commit/168f6a5a2717bc80961da6ba751509cd8c7ace0d ). These lines have remained the same at NOAA-GFDL's master branch up until now. However, the changes that Bill Ramstrom and I have made to the source relative to what was in the HAFS-community git repo did not include differences at these lines (363, 374, 379 at https://github.com/hafs-community/UFS_UTILS/commit/11b3a182a5a5407e7005a6b20f56473b8c524f99 ), which indicates to me that the code in the HAFS repo did not include Zhi Liang's changes in 2015 and thus may be out of date. Looking at the initial commit for make_hgrid.c in the HAFS repo, it looks like these lines were never changed there.

I'll modify the code I have to reflect the prior version and submit a new commit, along with changes to the print statements.

-Kyle